community-operators/rook/rook.v0.1.0.csv.k8s.yaml

```diff
@@ -0,0 +1,306 @@
+        },
+      ]
+spec:
+  displayName: rook
+  description: |
+      Rook: Storage Orchestration for Kubernetes
+
+      Rook runs as a cloud-native service for optimal integration with applications in need of storage, and handles the heavy-lifting behind the scenes such as provisioning and management.      ## Before Your Start
+      Rook orchestrates battle-tested open-source storage technologies including Ceph, which has years of production deployments and runs some of the worlds largest clusters.
+      Rook is open source software released under the Apache 2.0 license. Rook has an active developer and user community.
+
+  keywords: ['rook', 'ceph', 'storage', 'object storage', 'open source']
+  version: 0.1.0
+  minKubeVersion: 1.11.0
+  maturity: alpha
+  maintainers:
+  - name: Red Hat, Inc.
+    email: customerservice@redhat.com
+  provider:
+    name: Red Hat, Inc.
+  labels:
+    alm-owner-etcd: etcdoperator
+    operated-by: rookoperator
+  selector:
+    matchLabels:
+      alm-owner-etcd: etcdoperator
+      operated-by: rookoperator
+  links:
+  - name: Blog
+    url: https://blog.rook.io
+  - name: Documentation
+    url: https://rook.github.io/docs/rook/master/
+  - name: rook Operator Source Code
+    url: https://github.com/rook/rook/tree/master/pkg/operator/ceph
+
+  icon:
+  - base64data: #TODO
+    mediatype: image/png
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: CephCluster
+      name: cephclusters.ceph.rook.io
+      version: v1
+      displayName: Ceph Cluster
+      description: Represents a Ceph cluster.
+    - kind: CephBlockPool
+      name: cephblockpools.ceph.rook.io
+      version: v1
+      displayName: Ceph Block Pool
+      description: Represents a Ceph Block Pool.
+    - kind: CephFilesystem
+      name: cephfilesystems.ceph.rook.io
+      version: v1
+      displayName: Ceph Filesystem
+      description: Represents a Ceph Filesystem.
+    - kind: CephNFS
+      name: cephnfses.ceph.rook.io
+      version: v1
+      displayName: Ceph NFS
+      description: Represents a Ceph NFS interface.
+    - kind: CephObjectStore
+      name: cephobjectstores.ceph.rook.io
+      version: v1
+      displayName: Ceph Object Store
+      description: Represents a Ceph Object Store.
+    - kind: CephObjectStoreUser
+      name: cephobjectstoreusers.ceph.rook.io
+      version: v1
+      displayName: Ceph Object Store User
+      description: Represents a Ceph Object Store User.
+    - kind: Volume
+      name: volumes.rook.io
+      version: v1alpha2
+      displayName: Ceph Volume
+      description: Represents a Ceph volume.
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: true
+  - type: MultiNamespace
+    supported: false
+  - type: AllNamespaces
+    supported: true
+  install:
+    spec:
+      deployments:
+      - name: rook-ceph-operator
+        spec:
+          replicas: 1
+          selector: null # Not supported?
```

Comment:
Not sure I understand this comment.

Reply:
I think I got it from a discussion with @ecordell I don't exactly remember the rationale, perhaps he does?

Reply:
This block is a standard deployment spec - you need the selector for it to be valid.

Sorry, I don’t remember a conversation discussing otherwise 😅

Reply:
I think I remember, I got that generated by scorecard because our current deployment spec does not specify any selector and @ecordell told me that "null" in this case wasn't supported.